firstvoices/backend/utils/media_utils.py

```diff
@@ -0,0 +1,23 @@
+from django.core.exceptions import ObjectDoesNotExist
+
+from backend.models.media import Image, Video
+from backend.models.sites import Site
+
+
+def verify_media_source(site_slug, media_type, media_id):
```

Comment:
Suggestion: add a default for media_type (probably image?) and make the type options an enum